dm/openapi/spec/dm.yaml

```diff
@@ -1825,6 +1825,8 @@ components:
             - "full"
             - "incremental"
             - "all"
+            - "dump"
+            - "load&sync"
```

Comment:
no need to expose this one

Reply:
get it, i will revert relate code and test.

Reply:
done